x/posts/client/rest/tx.go

```diff
@@ -16,6 +16,7 @@ func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/posts/reactions", addReactionToPostHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/posts/reactions", removeReactionToPostHandler(cliCtx)).Methods("DELETE")
 	r.HandleFunc("/posts/{postID}/answers", addAnswerToPostPollHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/reaction", registerReactionHandler(cliCtx)).Methods("POST")
```

Comment:
Probably we should change it to be more similar to the query command: 

```suggestion
	r.HandleFunc("/posts/registeredReactions", registerReactionHandler(cliCtx)).Methods("POST")
```